daml/Marketplace/Issuance/CFI.daml

```diff
@@ -0,0 +1,45 @@
+module Marketplace.Issuance.CFI where
+
+-- |Classification of Finanical Instruments, denoted using ISO 10962:
+-- https://en.wikipedia.org/wiki/ISO_10962
+newtype CFI = CFI { code : Text } deriving (Eq)
```

Comment:
for sure!

interesting fact I just learned (thank you Wikipedia):

> ISO is not an acronym. ISO gives this explanation of the name: "Because 'International Organization for Standardization' would have different acronyms in different languages (IOS in English, OIN in French), our founders decided to give it the short form ISO. ISO is derived from the Greek word isos (ίσος, meaning "equal"). Whatever the country, whatever the language, the short form of our name is always ISO." During the founding meetings of the new organization, the Greek word explanation was not invoked, so this meaning may have been made coined later as a backronym.